packages/beacon-node/src/chain/chain.ts

```diff
@@ -5,7 +5,7 @@ import {
   CachedBeaconStateAllForks,
   computeEpochAtSlot,
   computeStartSlotAtEpoch,
-  createCachedBeaconState,
+  createFinalizedCachedBeaconState,
```

Comment:
i don't think we want this to be renamed to finalized because it has other connotations other than having a finalized pubkey maps